cfclient/cfapps/cache.go

```diff
@@ -32,7 +33,8 @@ func NewCache() *Cache {
 func (c *Cache) Start() {
 	go func() {
 		cacheDuration := app.Get().Config.GetDuration("FIREHOSE_CACHE_DURATION_MINS")
-		update := time.NewTicker(30 * time.Second).C
+		cacheUpdate := app.Get().Config.GetDuration("FIREHOSE_CACHE_UPDATE_INTERVAL_SECS")
```

Comment:
could you add a check here to make sure that users don't use a very low number and "destroy" their infrastructure ? I would always use the minimum between a default value (30 for example) and the value the user configured.

Reply:
```
    constraints:
      min: 30
```